core/application/impl/app_configuration_impl.cpp

```diff
@@ -149,24 +149,24 @@ namespace kagome::application {
 
   void AppConfigurationImpl::parse_blockchain_segment(rapidjson::Value &val) {
     std::string genesis_path_str;
-    load_str(val, "genesis", genesis_path_str);
+    load_str(val, "chain", genesis_path_str);
     genesis_path_ = fs::path(genesis_path_str);
   }
 
   void AppConfigurationImpl::parse_storage_segment(rapidjson::Value &val) {
     std::string base_path_str;
-    load_str(val, "base_path", base_path_str);
+    load_str(val, "base-path", base_path_str);
     base_path_ = fs::path(base_path_str);
   }
 
   void AppConfigurationImpl::parse_network_segment(rapidjson::Value &val) {
     load_ma(val, "listen-addr", listen_addresses_);
     load_ma(val, "bootnodes", boot_nodes_);
-    load_u16(val, "p2p_port", p2p_port_);
-    load_str(val, "rpc_http_host", rpc_http_host_);
-    load_u16(val, "rpc_http_port", rpc_http_port_);
-    load_str(val, "rpc_ws_host", rpc_ws_host_);
-    load_u16(val, "rpc_ws_port", rpc_ws_port_);
+    load_u16(val, "port", p2p_port_);
+    load_str(val, "rpc-host", rpc_http_host_);
+    load_u16(val, "rpc-port", rpc_http_port_);
+    load_str(val, "ws-host", rpc_ws_host_);
```

Comment:
Think about to rename "rpc-ws-" or "ws-rpc-"

Reply:
you mean rename variable name?

Reply:
First of all about options. So strange to mix mechanism (RPC) and protocol (Websocket).

Reply:
Agree, but this is how it is in substrate :(

Reply:
Support @xDimon here, it's weird and we don't have to follow substrate's issues, when we aim to provide similar interface.

Reply:
Our goal is to make it easy for substrate users to switch to Kagome. I suggest for now keep compatibility as much as possible